lb.tf

```diff
@@ -84,13 +84,13 @@ resource "aws_lb_target_group" "default" {
   }
 
   health_check {
-    interval            = 30
+    interval            = var.health_check != null ? var.health_check.interval : null
     timeout             = var.protocol != "TCP" ? 3 : null
     protocol            = var.protocol
-    path                = var.protocol != "TCP" ? var.health_check_path : null
+    path                = var.protocol != "TCP" ? (var.health_check != null ? var.health_check.path : null) : null
     matcher             = var.protocol != "TCP" ? 200 : null
-    healthy_threshold   = 3
-    unhealthy_threshold = 3
+    healthy_threshold   = var.health_check != null ? var.health_check.healthy_threshold : null
+    unhealthy_threshold = var.health_check != null ? var.health_check.unhealthy_threshold : null
```

Comment:
Defining and using a default object would make this much nicer...

Reply:
These can now be simplified to use just `var.health_check.healthy_threshold` and `var.health_check.unhealthy_threshold` as the object should always be there now...